app/assets/main/react_components/context/Footprint/StaticDataContext.js

```diff
@@ -0,0 +1,32 @@
+import React, { useContext } from "react";
+
+const TextsContext = React.createContext();
+export const useTexts = () => useContext(TextsContext);
+
+const LocaleContext = React.createContext();
+export const useLocaleData = () => useContext(LocaleContext);
+
+/**  To wrap components that need access to the context in a context provider */
+export const StaticDataProvider = ({ children, commonText, currency, lang, lifestyleFootprintsText, modelText, registrationsText, slug }) => {
+    
+    const texts = {
+        registrationsText: JSON.parse(registrationsText),
+        commonText: JSON.parse(commonText),
+        modelText: JSON.parse(modelText),
+        lifestyleFootprintsText: JSON.parse(lifestyleFootprintsText)
+    };
+    
+    const localeData = {
+        slug: slug,
+        lang: lang,
+        currency: currency
+    };
+
+    return (
+        <TextsContext.Provider value={texts}>
+            <LocaleContext.Provider value={localeData}>
```

Comment:
Måste dessa ligga "nestlade", altså en i den andra? Kan de "anropas" på samma rad? 
Spelar det någon roll vilken som är ytterst och innerst?

Reply:
Ja. {children} är de Reactkomponenter du placerar inuti i Container.jsx. Nestlas de inte så kan inte barnen placeras "längst in" och alltså ges tillgång till all den data som dessa skal tillhandahåller. 

Ordning spelar ingen roll, de pratar ändå inte "med varandra" utan är bara till för att alla children ska ha tillgång till all data.

"Anropas på samma rad" är jag osäker på vad du menar med =)